public/scripts/extensions/caption/settings.html

```diff
@@ -54,36 +54,36 @@
                         <option data-type="openrouter" value="google/gemini-pro-vision">google/gemini-pro-vision</option>
                         <option data-type="openrouter" value="google/gemini-flash-1.5">google/gemini-flash-1.5</option>
                         <option data-type="openrouter" value="liuhaotian/llava-yi-34b">liuhaotian/llava-yi-34b</option>
-                        <option data-type="ollama" value="ollama_current">[Currently selected]</option>
+                        <option data-type="ollama" value="ollama_current" data-i18n="currently_selected">[Currently selected]</option>
                         <option data-type="ollama" value="bakllava:latest">bakllava:latest</option>
                         <option data-type="ollama" value="llava:latest">llava:latest</option>
-                        <option data-type="llamacpp" value="llamacpp_current">[Currently loaded]</option>
-                        <option data-type="ooba" value="ooba_current">[Currently loaded]</option>
-                        <option data-type="koboldcpp" value="koboldcpp_current">[Currently loaded]</option>
-                        <option data-type="custom" value="custom_current">[Currently selected]</option>
+                        <option data-type="llamacpp" value="llamacpp_current" data-i18n="currently_loaded">[Currently loaded]</option>
+                        <option data-type="ooba" value="ooba_current" data-i18n="currently_loaded">[Currently loaded]</option>
+                        <option data-type="koboldcpp" value="koboldcpp_current" data-i18n="currently_loaded">[Currently loaded]</option>
+                        <option data-type="custom" value="custom_current" data-i18n="currently_selected">[Currently selected]</option>
                     </select>
                 </div>
                 <label data-type="openai,anthropic,google" class="checkbox_label flexBasis100p" for="caption_allow_reverse_proxy" title="Allow using reverse proxy if defined and valid.">
                     <input id="caption_allow_reverse_proxy" type="checkbox" class="checkbox">
-                    Allow reverse proxy
+                    <span data-i18n="Allow reverse Proxy">Allow reverse Proxy</span>
```

Comment:
Sorry but capitalization is still changed from the original

<img src="https://github.com/SillyTavern/SillyTavern/assets/18619528/717c30d9-523d-4765-bc53-9d2bfd9a54a5" width="200">

Reply:
fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/c67407e80b74d82d129a80140bfcc49ca76cce9d
ty for help!